crates/hdk/src/lib.rs

```diff
@@ -97,7 +90,10 @@
 //!
 //! And every host function defined by Holochain has a convenience wrapper in HDK that does the type juggling for you.
 //!
-//! ## Extern callbacks
+//! Low-level communication between the conductor and WASM binaries, like typing and serialization of data, is abstracted by the HDK.
+//! Using the HDK, hApp developers can focus on their application's logic. [Learn more about WASM in Holochain.](https://github.com/holochain/holochain/blob/develop/crates/hdk/ON-WASM.md)
+//!
+//! ## Extern callbacks = Zome functions
```

Comment:
```suggestion
//! ## External callbacks = Zome functions
```

The reason I'm suggesting changing 'extern' to 'external' here is that, technically, both zome functions and reserved guest callbacks are all externs, even marked up with `#[hdk_extern]` proc macro. Hopefully this disambiguates from the `extern` keyword and highlights that it's a more general definition of external.